control-plane-operator/controllers/hostedcontrolplane/oauth/config.go

```diff
@@ -128,3 +129,17 @@ func generateOAuthConfig(ctx context.Context, client crclient.Client, namespace
 	}
 	return serverConfig, nil
 }
+
+func configNamedCertificates(servingCerts []configv1.APIServerNamedServingCert) []configv1.NamedCertificate {
+	result := []configv1.NamedCertificate{}
+	for i, cert := range servingCerts {
```

Comment:
do we not need this here? https://github.com/openshift/hypershift/blob/d19eb9d8f9392c654020e8a9f7c66f469f2c677a/control-plane-operator/controllers/hostedcontrolplane/kas/config.go#L256-L262

Reply:
that's done in an earlier section but we can model it the same way.....